packages/sanity/src/core/tasks/plugin/TasksStudioActiveToolLayout.tsx

```diff
@@ -77,6 +77,7 @@ function TasksStudioActiveToolLayoutInner(props: ActiveToolLayoutProps) {
         {isOpen && (
           <SidebarMotionLayer
             animate="visible"
+            zOffset={1000}
```

Comment:
@pedrobonamin 👋 Quick question: which issue is this trying to solve? 👀

Reply:
👋🏻 
<img width="604" alt="Screenshot 2024-04-17 at 15 47 19" src="https://github.com/sanity-io/sanity/assets/46196328/667784f5-3970-4af9-88cc-64c8329796c2">

Reply:
Some elements are showing on top of the tasks sidebar.